geode-core/src/distributedTest/java/org/apache/geode/distributed/internal/P2PMessagingConcurrencyDUnitTest.java

```diff
@@ -341,6 +450,10 @@ private static Properties gemFireConfiguration(
      */
     props.setProperty("conserve-sockets", String.valueOf(conserveSockets));
 
+    if (TEST_WITH_TLS_KEY_UPDATE_MESSAGE_PROCESSING) {
```

Comment:
Make a separate test or parameter to this test.

Reply:
reverted changes to this class